test/adapters/http/test-server.js

```diff
@@ -343,6 +343,74 @@ class TestHTTPServer {
                 }
             });
 
+            this.app.get('/pageByRequest', (req, res) => {
+                // returns 5 pages with 1 point at a time and provides the
+                // `Link` header needed to get the subsequent elements
```

Comment:
This route if for paging using `-paramUnit 'request'` and not the one you're looking for which is [here](https://github.com/juttle/juttle/pull/435/files#diff-0835c78cfbeb374688a7590ad92c8564R401) and used when you use the `-followLinkHeader`.